metricbeat/module/haproxy/_meta/docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+== haproxy Module
+
+This is the haproxy Module.  To enable stats collection from HAProxy, you must enable the stats socket via TCP.  
+For example, placing the following statement under the `global` or `default` section of the haproxy config:
+
+`stats socket 0.0.0.0:14567`
```

Comment:
I recommend providing a more secure example. Either show an example that binds only to `127.0.0.1` or use a unix socket.